octree_web_viewer/src/bin/points_web_viewer.rs

```diff
@@ -96,7 +96,7 @@ fn main() {
             })
     })
     .bind(&ip_port)
-    .expect(&format!("Can not bind to {}", &ip_port))
+    .unwrap_or_else(|_| panic!("Can not bind to {}", &ip_port))
```

Comment:
from https://rust-lang.github.io/rust-clippy/master/#expect_fun_call
> The function will always be called.

and
> If the function has side-effects, not calling it will change the semantic of the program, but you shouldn’t rely on that anyway.

Reply:
makes sense, thanks